webapp/src/components/legal_hold_table/legal_hold_row/legal_hold_row.tsx

```diff
@@ -27,15 +30,128 @@ const LegalHoldRow = (props: LegalHoldRowProps) => {
         props.releaseLegalHold(lh);
     };
 
-    const usernames = props.users.map((user) => {
-        if (user) {
-            return `@${user.username} `;
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const [message, setMessage] = useState('');
+    const [error, setError] = useState('');
+
+    const bundleLegalHold = async (legalHold: LegalHold) => {
+        try {
+            await Client.bundleLegalHold(legalHold.id);
+            lh.locks?.push('bundle');
+        } catch (err) {
+            if ('message' in (err as Error)) {
+                setError((err as Error).message);
+            }
         }
-        return 'loading...';
-    });
+    };
+
+    let statusMessage: React.ReactNode | undefined;
+    if (error) {
+        statusMessage = (
+            <StatusMessage
+                state='warn'
+                message={error}
+            />
+        );
+    } else if (message) {
+        statusMessage = (
+            <StatusMessage
+                state='success'
+                message={message}
+            />
+        );
+    }
 
     const downloadUrl = Client.downloadUrl(lh.id);
 
+    const downloadButton = (
+        <OverlayTrigger
+
+            // @ts-ignore
+            delayShow={300}
+            placement='top'
```

Comment:
What typescript error is occurring here?

Reply:
```
Type '{ children: Element; delayShow: number; placement: "top"; overlay: Element; }' is not assignable to type 'IntrinsicAttributes & OverlayTriggerProps & { disabled?: boolean | undefined; className?: string | undefined; } & RefAttributes<any>'.
  Property 'delayShow' does not exist on type 'IntrinsicAttributes & OverlayTriggerProps & { disabled?: boolean | undefined; className?: string | undefined; } & RefAttributes<any>'.ts(2322)
```

Reply:
If I remember correctly, this ts error is somehow spurious - the props do exist and influence UI behaviour even though ts says they don't. It was a while ago though, so may be misremembering.